Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -46,7 +46,7 @@
         <PathfindingWeight>1</PathfindingWeight>
         <Width>1</Width>
         <Height>1</Height>
-        <LinksToNeighbours>false</LinksToNeighbours>
+        <LinksToNeighbours></LinksToNeighbours>
```

Comment:
LinksToNeighbours isn't a required tag. This was left in there in case we decide that doors should be a part of the logic. I can remove that if you'd like.